ble_controller/Kconfig

```diff
@@ -23,14 +37,15 @@ config BT_LL_NRFXLIB
 	select BT_CTLR_EXT_SCAN_FP_SUPPORT
 	select BT_CTLR_PHY_UPDATE_SUPPORT
 	select BT_CTLR_CHAN_SEL_2_SUPPORT
-	# This controller only supports nRF52, it does not support 51 or
+	# This controller only supports nRF52 and nRF53, it does not support 51 or
 	# 91 (91 doesn't even have the physical HW needed for BLE).
-	depends on SOC_SERIES_NRF52X
+	depends on (SOC_SERIES_NRF52X || SOC_NRF5340_CPUNET)
```

Comment:
This needs to be changed to SOC_SERIES_NRF53X.
Not only it complies with the SOC_SERIES_NRF52X, but also, it complies with line 72.

Reply:
You cannot run the controller on the app core. I should rather update this on line 72.